src/main/java/hudson/plugins/parameterizedtrigger/BlockableBuildTriggerConfig.java

```diff
@@ -74,7 +78,7 @@ protected QueueTaskFuture schedule(AbstractBuild<?, ?> build, Job project, List<
                 // which is most likely unintended. Might make sense to do it at BuildTriggerConfig for all.
                 list = CollectionUtils.immutableList(list, new DifferentiatingAction());
 
-                
+
```

Comment:
FYI

Reply:
😅 thanks!

Reply:
Quick general question: 
Correct me if I'm wrong but ideally, there should be just one line after line 79, but currently, there are 2 empty lines. 
So, we don't want to correct it because we don't want to make any changes in the original code just for the sake of keeping things the way they are unless it's really urgent. Is this what our intention is?

Reply:
Understood :+1: 
also wow, the comments on the PR you shared above, are kind of intense. I get your point now.